synapse/models/material.py

```diff
@@ -31,37 +31,88 @@ class MatModule(s_module.CoreModule):
 
     def getModelDefs(self):
         modl = {
+
+            'interfaces': (
+
+                ('phys:object', {
```

Comment:
Would we want to include an :asof prop?

Reply:
Not sure it makes sense on one-time-knowledge forms which will inherit this interface. Anything like vitals will def include a `:time` but open to discussion.